source/FreeRTOS_DNS.c

```diff
@@ -315,7 +313,6 @@
 
         if( pxInfo != NULL )
         {
-            configASSERT( pxLastInfo != pxInfo );
```

Comment:
Why did we remove this?

Reply:
`pxLastInfo` is an unused variable, we are removing it.